### PR TITLE
chore(cleanup): rename GUI to AntiMicroX, remove antimicroX references, repo rename

### PR DESCRIPTION
**Pull request by gombosg**
For #26 

## Proposed changes 

Magic regexps were:
`(?<!juliagoda\/)antimicroX`
Also: `/* antimicroX` for code comments.

Repo has to be renamed after merge to `antimicrox`!

Renamed GUI references to `AntiMicroX`.

Tested:
- local build
- GUI in general
- `desktop-file-validate /usr/share/applications/io.github.antimicrox.antimicrox.desktop`
- `appstream-util validate-relax /usr/share/metainfo/io.github.antimicrox.antimicrox.appdata.xml`
- `man antimicrox`
- desktop icon
- setting migration

Not touched:
- changelog
- `juliagoda/antimicroX` links and similar references to old repo
- Fedora package name
- Legacy config path folders

### CODE REVIEW

**Comment by OlegPopow** [comment]
What is the point of some changes? Seriously, I don't understand - the old repository has not changed its name. Even if you type a lowercase name, it will automatically change to a capital letter in your browser.

img

The same applies here:

img

Anyway, you've changed the case of your repository everywhere except for your own - of course, I mean the title.

It looks a bit incomprehensible and inconsistent - if you change the title with such persistence, the name of the repository would also need to be changed.

**Comment by gombosg** [comment]
Thanks for the review!
Yes, as I wrote in the PR description, this repo will be renamed to `antimicrox` after merging.
In the next PR, the dockerfiles will be fixed (there I made a mistake of renaming the old repo URL, which is useless at the moment).

The rename is because we don't want 3 names to be present (`antimicroX`, `antimicrox` and `AntiMicroX`), only 2: `antimicrox` for the repo, binary name etc. and `AntiMicroX` in the GUI.

Hence, I'm getting rid of `antimicroX` references here.

File: src/qkeydisplaydialog.ui [hunk 1]
```diff
@@ -38,7 +38,7 @@
...
...&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press a key on your keyboard to see how the key is detected by this application. The window will show the system native key value, the original value given by Qt (if applicable), and the custom value used by AntiMicroX.&lt;/p&gt;&lt;p&gt;The AntiMicroX key value and the Qt key value will usually be the same. AntiMicroX tries to use the key values defined in Qt when possible. Check the page &lt;a href=&quot;http://doc.qt.io/qt-5/qt.html#Key-enum&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;http://doc.qt.io/qt-5/qt.html#Key-enum&lt;/span&gt;&lt;/a&gt; for a list of values defined by Qt. If you discover that a key is not natively supported by this program, please report the problem to AntiMicroX's &lt;a href=&quot;https://github.com/AntiMicroX/antimicrox/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;GitHub page&lt;/span&gt;&lt;/a&gt; so that the program can be edited to support it directly. As it is, a custom prefix is added to unknown values so they can still be used; the main problem is that the profile will no longer be portable.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (pktiuk) [inline src/qkeydisplaydialog.ui:41]: Strange, this commit doesn't seem to change anything

**Review by pktiuk — APPROVED** [review]

File: BUILDING.md [hunk 2]
```diff
@@ -21,10 +21,10 @@ installed on your system in order to build this program:
 
 ### Building Under Linux
 
-In order to build this program, open a terminal and cd into the antimicroX
+In order to build this program, open a terminal and cd into the antimicrox
 directory. Enter the following commands in order to build the program:
 
-    cd antimicroX
+    cd antimicrox
```
[the remark this answers is not in the document]
Comment (pktiuk) [reply BUILDING.md:27]: I will do it on branch: `general_cleanup`

File: src/qkeydisplaydialog.ui (hunk 1 above)
Comment (pktiuk) [reply src/qkeydisplaydialog.ui:41]: I see this change now

**Pull request merged**